frontend/portals/HistorySuggestions/index.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { I18n } from '@shopgate/engage/components';
+import { themeConfig } from '@shopgate/pwa-common/helpers/config';
+import Button from '@shopgate/pwa-ui-shared/Button';
+import { isIOSTheme } from '@shopgate-ps/pwa-extension-kit/env/helpers';
+import { css } from 'glamor';
+import classnames from 'classnames';
+import connect from './connector';
+import SuggestionList from './components/SuggestionList';
+import stylesFile from './style';
+
+const { colors } = themeConfig;
+const isIOS = isIOSTheme();
+
+const styles = {
+  deleteHistory: isPersistentSearchBar => css({
+    textDecoration: 'underline',
+    marginLeft: isIOS || isPersistentSearchBar ? 46 : 72,
+    marginTop: 10,
+    color: colors.shade3,
+    fontSize: 14,
+  }).toString(),
+};
+
+/**
+ * @return {JSX}
+ */
+const HistorySuggestions = ({
+  onClick,
+  searchHistory,
+  suggestions,
+  searchPhrase,
+  children,
+  deleteSearchHistory,
+  visible,
+  bottomHeight,
+  name,
+}) => {
+  const isPersistentSearchBar = name === 'persistent-search-bar.search.suggestions';
+  if (!visible || (suggestions && suggestions.length && searchPhrase !== '') || !searchHistory.length) {
+    return children;
+  }
+
+  /**
+   * @param {Event} e Event
+   * @param {string} searchTerm searchTerm
+   */
+  const handleClick = (e, searchTerm) => {
+    e.currentTarget.value = searchTerm;
+    onClick(e, searchTerm);
+  };
+
+  return (
```

Comment:
This block also renders when search suggestions are fetched. Its visible with slow networks